### PR TITLE
pmap->jit transition + misc updates accumulated over time

### PR DESCRIPTION
**Pull request by akolesnikoff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,23 +75,30 @@ codebase:
...
 - [Sigmoid Loss for Language Image Pre-Training](https://arxiv.org/abs/2303.15343), by
-  Xiaohua Zhai*, Basil Mustafa, Alexander Kolesnikov, Lucas Beyer*\
-  Resources: [colab and models](https://colab.research.google.com/github/google-research/big_vision/blob/main/big_vision/configs/proj/image_text/SigLIP_demo.ipynb), code TODO.
+  Xiaohua Zhai*, Basil Mustafa, Alexander Kolesnikov, Lucas Beyer*.
+- [A Study of Autoregressive Decoders for Multi-Tasking in Computer Vision](https://arxiv.org/abs/2303.17376), by
+  Lucas Beyer*, Bo Wan*, Gagan Madan*, Filip Pavetic*, Andreas Steiner*, Alexander Kolesnikov, André Susano Pinto, Emanuele Bugliarello, Xiao Wang, Qihang Yu, Liang-Chieh Chen, Xiaohua Zhai*.
+- [Image Captioners Are Scalable Vision Learners Too](https://arxiv.org/abs/2306.07915), by
+- [Tuning computer vision models with task rewards](https://arxiv.org/abs/2302.08242), by
+  André Susano Pinto*, Alexander Kolesnikov*, Yuge Shi, Lucas Beyer, Xiaohua Zhai.
+  Michael Tschannen*, Manoj Kumar*, Andreas Steiner*, Xiaohua Zhai, Neil Houlsby, Lucas Beyer*.
```
Comment (andsteing) [inline README.md:84]: I think the Cappa paper link is missing here

Comment (andsteing) [reply README.md:84]: Oh, it's one line above.

So I think line 84 should be moved above line 82 to make things right.

Comment (andsteing) [reply README.md:84]: Oh, it's one line above.

So I think line 84 should be moved above line 82 to make things right.

**Review by lucasb-eyer — APPROVED** [review]
Overall LGTM, but it's hard to tell if a file is missing or not. For example, do we want to add the deprecated trainer too, or not?

**Review by lucasb-eyer — APPROVED** [review]
Scrolled through it and found no remaining major issues. Given that you actually tested it on Cloud, OK to submit.

**Pull request merged**